compose/.apps/rustdesk/rustdesk.yml

```
@@ -0,0 +1,18 @@
+services:
+  rustdesk:
+    container_name: rustdesk
+    environment:
+      - PGID=${PGID}
+      - PUID=${PUID}
+      - TZ=${TZ}
+      - RELAY=${RUSTDESK_RELAY_SERVER}
```

```suggestion
      - RELAY=${RUSTDESK_RELAY_SERVER}
      - TZ=${TZ}
```
Sort these alphabetically please.